Carbon.Domain.EntityFrameworkCore.Extensions/EFExtensions.cs

```diff
@@ -596,9 +597,20 @@ public static IQueryable<T> WhereContains<T>(this IQueryable<T> query, Expressio
                 }
                 else
                 {
-                    ConstantExpression constantExpression = Expression.Constant(value.ToLower(culture));
-                    MethodCallExpression methodCallExpression = Expression.Call(instance2, "Contains", Type.EmptyTypes, constantExpression);
-                    expression = ((expression != null) ? ((Expression)Expression.OrElse(expression, methodCallExpression)) : ((Expression)methodCallExpression));
+                    if (useTrEnSearch)
```

Comment:
Parameter naming must be fixed; one case uses "Contains" operator, other one uses "Equals" operator. Method naming indicates that, its a "Contains" operator method but if "useTrEnSearch = false", it becomes "Equals" operator method and thats missmaches with the method naming